dhis-2/dhis-e2e-test/src/test/java/org/hisp/dhis/tracker/importer/TrackerExportTests.java

```diff
@@ -284,4 +292,52 @@ public void shouldReturnAscOrderedEventByTEIAttribute()
         List<String> events = response.extractList( "instances.event.flatten()" );
         assertEquals( List.of( "ZwwuwNp6gVd", "olfXZzSGacW" ), events, "Events are not in the correct order" );
     }
+
```

Comment:
Maybe these 3 tests could be one parameterized test.
It would more easily show input and expected output